### DISCUSSION TITLE
Copyright

### DISCUSSION THREAD

**Discussion by KWodarczyk** [microsoft/CsWin32#971] [General]
Hi where can I find the copyright notice for this ?
I have to include MIT license  and the copyright notice.

e.g.

Copyright (c) Microsoft Corporation. All right reserved.
https://github.com/microsoft/CsWin32/blob/main/LICENSE

**Comment by AArnott** [comment]
The copyright is templated in stylecop.json and you can find it at the top of every source file. e.g.

https://github.com/microsoft/CsWin32/blob/370b297d10403179a5cc0cf29f536b129b31bf01/src/Microsoft.Windows.CsWin32/CustomAttributeTypeProvider.cs#L1-L2

**Reply by KWodarczyk** [reply]
Ok but how anyone would know that by just looking at the repo ?

**Reply by AArnott** [reply]
I'll add a COPYRIGHT file to the repo. I just wanted to give you your answer more quickly.

**Reply by KWodarczyk** [reply]
Sure. Thank you.
